pkg/reconciler/autoscaling/hpa/keda_hpa.go

```diff
@@ -42,7 +46,10 @@ import (
 	"knative.dev/autoscaler-keda/pkg/reconciler/autoscaling/hpa/resources"
 )
 
-const allActivators = 0
+const (
+	allActivators                    = 0
+	KedaAutoscaleAnotationAutocreate = autoscaling.GroupName + "/scaled-object-auto-create"
```

Comment:
`KedaAutoscaleAnnotationAutocreate`

Reply:
I see the same spelling error elsewhere. I am going to fix it.